### ISSUE TITLE
Help with training model for noise reduction of darker area lines

### ISSUE THREAD

**Issue by CarbonPool** [nagadomi/waifu2x#405]
For most of the darker images similar to this, I have tried all three-level noise reduction models, and their lines are smeared. What training set is suitable to help preserve these details?

- origin
![sec](https://user-images.githubusercontent.com/26106045/133475826-259190d4-9b42-4ac0-990d-8cba3ae27fcc.png)

- default cunet level3 2x
![sec(CUnet)(noise_scale)(Level3)(x2 000000)](https://user-images.githubusercontent.com/26106045/133476580-603b1be6-d0f6-431d-9f1a-ed7c6db53e1f.png)

-trained cunet model level3 2x
![sec(MyCunet_v2)(noise_scale)(Level3)(x2 000000)](https://user-images.githubusercontent.com/26106045/133476655-9c862f7c-0794-4726-961d-e5e80201800b.png)

**Comment by nagadomi** [comment]
For training data, you can add an image with a narrow range of colors.
Another way is to normalize the input image to be minimum 0 and maximum 255, convert it with waifu2x, and then revert the color range.
